fakenet/fakenet.py

```diff
@@ -196,11 +197,28 @@ def start(self):
                     self.logger.error('Error starting %s listener:', listener_config['listener'])
                     self.logger.error(" %s" % e)
 
-
         # Start the diverter
         if self.diverter:
             self.diverter.start()
 
+        for listener in self.running_listener_providers:
+
+            # Only listeners that implement acceptListeners(listeners) 
+            # interface receive running_listener_providers
+            try:
+                #if callable(getattr(listener_provider, 'acceptListeners')):
+                listener.acceptListeners(self.running_listener_providers)
+            except:
```

Comment:
This would require modifying all listeners, and requiring users to account for this when creating custom listeners. How about catching AttributeError, which will automatically occur if the listener does not implement the interface?